src/controllers/fileDestinationUrl.controller.ts

```diff
@@ -0,0 +1,25 @@
+import type { Request, Response } from 'express';
+import { fileDestinationUrlService } from '../services';
+
+interface CreateFileDestinationRequest extends Request {
+  body: {
+    bucket: string;
+    fileName: string;
+    fileType: string;
+    maxSize: number;
+    path: string;
+  };
+}
+
+const createFileDestinationUrl = (
+  req: CreateFileDestinationRequest,
+  res: Response,
+): void => {
+  fileDestinationUrlService.createFileDestinationUrl(req.body)
+    .then((data) => res.json(data))
+    .catch((err) => res.status(500).json(err));
```

Comment:
This isn't capturing errors as I'd expect; I'm just getting empty objects in the response.

...[what an unexpectedly complicated situation](https://stackoverflow.com/questions/18391212/is-it-not-possible-to-stringify-an-error-using-json-stringify)!

Seems like [serialize-error](https://www.npmjs.com/package/serialize-error) might be the simplest solution? Assuming we do in fact want to publish our uncaught errors.

Reply:
Oh interesting!  I swore it was posting errors but maybe that was an earlier draft of the code (oh hey tests!)

Will dig in.

Reply:
Rather than import a dependency I just made a little utility method that will serialize the error.  This has the added benefit of stripping the stack since I don't think we want to expose that to API clients anyway.